azure-pipelines.yml

```diff
@@ -18,6 +18,12 @@ stages:
 - stage: build
   displayName: Build
   jobs:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - template: /eng/common/templates/job/onelocbuild.yml
+      parameters:
+        CreatePr: false
+        LclSource: lclFilesfromPackage
+        LclPackageId: 'LCL-JUNO-PROD-SYMREADER'
```

Comment:
Shouldn't this be SYMREADER-CONVERTER (we also have dotnet/symreader and dotnet/symreader-portable repos that presumably require distinct ids)?

Reply:
These are controlled by the loc team and I think there's a length limit on them? in any case, symreader and symreader-portable don't actually loc so that's why there's no package name collision as of now. if that changes in the future, I'll make sure that they adjust accordingly.